apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -24,6 +24,11 @@ import {
   asyncLoadCoteacherInvite,
   hiddenPlSectionIds,
 } from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
+import {queryParams, updateQueryParam} from '../../utils';
+import shapes from '@cdo/apps/templates/studioHomepages/shapes';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import DCDO from '@cdo/apps/dcdo';
```

Comment:
Why is this imported?

Reply:
Oh whoops this was from a merge conflict when pulling in staging and I missed removing this line. I'll fix this!